src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/UserRepositoryTests.kt

```diff
@@ -34,6 +34,19 @@ class UserRepositoryTests : CleanDatabaseTests()
         assertThat(nullResult).isNull()
     }
 
+    @Test
+    fun `gets user emails in alphabetical order`()
```

Comment:
Is this test name misleading? I can't see ordering code, and I didn't think containsExactlyElementsOf checked order.

Reply:
`containsExactlyElementsOf` is order sensitive. Seems sqlite returns the records in alphabetical order, hence no extra ordering logic.